diff --git a/index.php b/index_new.php
index 3975925..515f80f 100644
--- a/index.php
+++ b/index_new.php
@@ -22,4 +22,17 @@ foreach($users as $user) {
     echo '<br>';
 }
 
+echo '<br>';
+echo '<br>';
+
+$sql = "SELECT COUNT(*) AS jumlah_user FROM users";
+$result = $SQL -> query($sql);
+
+if ($result -> num_rows > 0) {
+    $row = $result -> fetch_assoc();
+    echo "Jumlah User : " . $row["jumlah_user"];
+} else {
+    echo "Jumlah User : 0";
+}
+
 ?>
\ No newline at end of file
